Source/JavaScriptCore/wasm/WasmBBQJIT.cpp

```diff
@@ -1684,16 +1687,16 @@ class BBQJIT {
         case TypeKind::Func:
         case TypeKind::Funcref:
         case TypeKind::Externref:
+        case TypeKind::Eqref:
             return storeOpForTypeKind(Types::IPtr.kind);
         case TypeKind::V128:
             RELEASE_ASSERT_NOT_REACHED_WITH_MESSAGE("Should be handling vectors separately (use V128Store, not a StoreOpType)");
             return StoreOpType::I64Store;
         case TypeKind::Void:
             RELEASE_ASSERT_NOT_REACHED();
             return StoreOpType::I64Store;
-        default:
-            RELEASE_ASSERT_NOT_REACHED();
         }
+        RELEASE_ASSERT_NOT_REACHED();
```

Comment:
I meant to add this in https://github.com/WebKit/WebKit/pull/11268 but missed this line.